tools/code-owners-parser/ci.yml

```diff
@@ -23,14 +23,7 @@ pr:
       - eng/common/scripts/get-codeowners.ps1
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml
+  template: /eng/common/pipelines/templates/stages/archetype-sdk-tool-pwsh.yml
   parameters:
-    ToolDirectory: tools/code-owners-parser
-    TestPostSteps:
-      - task: PowerShell@2
-        displayName: Test on code owner script
-        inputs:
-          pwsh: true
-          filePath: eng/common/scripts/get-codeowners.ps1
-          arguments: >
-            -Test
\ No newline at end of file
+    TargetDirectory: eng/common-tests/
```

Comment:
TODO: need to point to `eng/common-tests/codeowners`; but first need to move the .ps1 files to appropriate subdirs (both in common and common-tests)